tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AWS/SNS/PublishBatchAsyncIntegration.cs

```diff
@@ -0,0 +1,79 @@
+// <copyright file="PublishBatchAsyncIntegration.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.ComponentModel;
+using System.Threading;
+using Datadog.Trace.ClrProfiler.CallTarget;
+using Datadog.Trace.DuckTyping;
+using Datadog.Trace.Tagging;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.AWS.SNS
+{
+    /// <summary>
+    /// AWSSDK.SNS PublishBatchAsync CallTarget instrumentation
+    /// </summary>
+    [InstrumentMethod(
+        AssemblyName = "AWSSDK.SimpleNotificationService",
+        TypeName = "Amazon.SimpleNotificationService.AmazonSimpleNotificationServiceClient",
+        MethodName = "PublishBatchAsync",
+        ReturnTypeName = "System.Threading.Tasks.Task`1[Amazon.SimpleNotificationService.Model.PublishBatchResponse]",
+        ParameterTypeNames = new[] { "Amazon.SimpleNotificationService.Model.PublishBatchRequest", ClrNames.CancellationToken },
+        MinimumVersion = "3.0.0",
+        MaximumVersion = "3.*.*",
+        IntegrationName = AwsSnsCommon.IntegrationName)]
+    [Browsable(false)]
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class PublishBatchAsyncIntegration
+    {
+        private const string Operation = "PublishBatch";
+
+        /// <summary>
+        /// OnMethodBegin callback
+        /// </summary>
```

Comment:
nit: IMO these comments don't add anything, so I don't bother with them personally. I know we have them a lot of other places tho, so just an FYI! 😄

Reply:
Good to know for the future! Thanks!